crud/select/plan.lua

```diff
@@ -48,7 +49,8 @@ local function get_index_for_condition(space_indexes, space_format, condition)
     end
 end
 
-local function extract_sharding_key_from_scan_value(scan_value, scan_index, sharding_index)
+local function extract_sharding_key_from_scan_value(scan_value, scan_index, sharding_index,
+                                                    space_format, space_name, box_obj)
```

Comment:
I see
```lua
    if scan_index.id == sharding_index.id then
        return scan_value
    end
```
Is this code still relevant?

Reply:
as we discussed below I keep `extract_sharding_key_from_scan_value()` without changes as is.

Reply:
seems yes